WordPressShared/WordPressShared/Core/Views/WPNUXUtility.m

```diff
@@ -110,7 +110,7 @@ + (NSDictionary *)titleAttributesWithColor:(UIColor *)color {
     paragraphStyle.lineHeightMultiple = 0.9;
     paragraphStyle.lineBreakMode = NSLineBreakByWordWrapping;
     paragraphStyle.alignment = NSTextAlignmentCenter;
-    NSDictionary *attributes = @{NSFontAttributeName: [WPNUXUtility titleFont],
+    NSDictionary *attributes = @{NSFontAttributeName: [UIFont preferredFontForTextStyle:UIFontTextStyleTitle2],
```

Comment:
If you merge `develop` into this branch I think you could get that font, with a maximum size, from `[WPStyleGuide fontForTextStyle]`. I am not sure that's necessary, or even if it makes sense.

Reply:
It's doesn't seem to be possible to `#import "WordPress-Swift.h"` in WorldPressShared, so that is not available for us at that point :( 

But I was thinking about making a set of `dynamic fonts` (and maybe to mark the ones declared in `WPFontManager` as static fonts somehow) so we can use them in places like this. And it becomes more obvious that if we use a static font, we will need to manually update it for size changes. This name changes might also help to avoid future issues with dynamic types.

Reply:
Oh, I missed those constraints, sorry! 😊